week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -38,7 +38,7 @@ function submit() {
     return false;
```

Comment:
Only one of the acceptance criteria has been missed out. 
> If any of the information is missing it shouldn't add the book and should show an alert
 In the if statement at line 31 author needs to be checked too e.g.:
```
  if (
    title.value == null ||
    title.value == "" ||
    author.value == null ||
    author.value == "" ||
    pages.value == null ||
    pages.value == ""
  ) {
```